content/workshops/tsp1-neuromorphic-chip-chris-eliasmith/index.md

```diff
@@ -0,0 +1,72 @@
+---
+
+## About This Workshop
+
+Join us for an exciting workshop featuring Dr. Chris Eliasmith as he presents the TSP1 neuromorphic chip, a cutting-edge hardware platform developed by Applied Brain Research. This event will provide insights into how neuromorphic computing can bridge the gap between artificial intelligence and biological neural systems.
+
+## What You'll Learn
+
+In this workshop, Dr. Eliasmith will cover:
+
+- **The TSP1 Architecture**: An overview of the TSP1 chip's unique design and capabilities
+- **Brain-Inspired Computing**: How the TSP1 embodies principles from neuroscience to create efficient, low-power computing solutions
+- **Real-World Applications**: Practical use cases where neuromorphic hardware like TSP1 excels, including edge computing, robotics, and adaptive systems
+- **Integration with Nengo**: How the TSP1 chip works seamlessly with the Nengo neural modeling framework
+- **Performance and Efficiency**: Comparisons with traditional computing architectures and insights into power consumption and speed
+
+## About the TSP1 Chip
+
+The TSP1 (Temporal Semantic Pointer 1) is a neuromorphic processor designed to efficiently implement the Neural Engineering Framework (NEF) and Semantic Pointer Architecture (SPA). Developed by Applied Brain Research, the TSP1 chip represents a significant advancement in brain-inspired computing hardware, offering:
+
+- **Ultra-low power consumption** suitable for edge deployment
+- **Real-time processing** of complex neural computations
+- **Scalable architecture** for building large-scale brain models
+- **Native support** for temporal dynamics and structured representations
+
+This hardware platform enables researchers and developers to deploy sophisticated cognitive models and neural networks in real-world applications where power efficiency and real-time performance are critical.
+
+## Who Should Attend
+
+This workshop is ideal for:
+
+- Researchers in neuromorphic computing and computational neuroscience
+- Engineers working on edge AI and embedded systems
+- Developers interested in brain-inspired computing platforms
+- Students exploring neuromorphic hardware and neural modeling
+- Anyone curious about the future of efficient AI computing
+
+## Prerequisites
+
+No specific prerequisites are required, though familiarity with neural networks and basic neuroscience concepts will enhance your understanding. Prior experience with Nengo is helpful but not necessary.
+
+## Resources
+
+- [Applied Brain Research Technology Page](https://www.appliedbrainresearch.com/technology)
+- [Nengo Neural Simulator](https://www.nengo.ai/)
```

Comment:
We should link to the internal /neuromorphic-computing/software/snn-frameworks/nengo/